telemetry/telemetry_record.py

```diff
@@ -73,15 +100,48 @@ def move_wpr_files(name_schema):
     """
     remote_data_path = os.path.join(CHROMIUM_SRC, 'tools/perf/page_sets/data/',
             name_schema)
-    local_path = os.path.join(os.getcwd(), 'data/')
+    local_path = os.path.join(os.getcwd(), 'data/wpr_source')
     p = Popen('mv {0} {1}'.format(remote_data_path, local_path), shell=True)
 
+def make_modified_json(num_urls):
+    """Creates *.json files to point to the right archive files
+
+    Located in ~/src/tools/perf/page_sets/data
+    :param num_urls: int for the number of target urls
+    TODO: add sha1?
+    """
+    json_path = '/home/jamshed/src/tools/perf/page_sets/data/'
+    json_template = ('{{'
+            '"description": "Describes the Web Page Replay archives for a user '
+            'story set. Dont edit by hand! Use record_wpr for updating.", '
+            '"archives": {{ "url{0}_page_set_000.wpr": ["url{0}"]}}}}\n')
+    for i in range(num_urls):
+        modified_name = '999999' + str(i)
+        file_name = 'url{0}_page_set.json'
+        with open('{0}{1}'.format(json_path,
+            file_name.format(modified_name)), 'wb') as f:
+            f.write(json_template.format(modified_name))
+
 def modify_wpr():
-    """Sets cacheable objects delay time to 0, creates  pc.wpr files"""
-    wpr_directory = os.path.join(os.getcwd(), 'data/')
+    """Sets cacheable objects delay time to 0, creates pc.wpr files
+
+    Note: *.pc.wpr files are converted to 999999 + str(url name), which lets
+    them be treated like regular urls. Just know that six 9's indicate a
+    modified wpr file
+    TODO: Fix the 999999 notation
```

Comment:
I was curious though: do the benchmark's themselves (python classes) really
need a URL in the filename? Could we just do track the current test URL in
memory, rather than on disk?

On Fri, Apr 17, 2015 at 10:19 PM, Jamshed Vesuna <notifications@github.com>
wrote:

> In telemetry/telemetry_record.py
> <https://github.com/colin-scott/page_load_time/pull/2#discussion_r28641546>
> :
>
> >  def modify_wpr():
> > -    """Sets cacheable objects delay time to 0, creates  pc.wpr files"""
> > -    wpr_directory = os.path.join(os.getcwd(), 'data/')
> > +    """Sets cacheable objects delay time to 0, creates pc.wpr files
> > +
> > +    Note: *.pc.wpr files are converted to 999999 + str(url name), which lets
> > +    them be treated like regular urls. Just know that six 9's indicate a
> > +    modified wpr file
> > +    TODO: Fix the 999999 notation
>
> Sorry, I posted that before seeing your comment. Right, the file name for
> the benchmark results don't need the url name (and currently don't).
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/colin-scott/page_load_time/pull/2/files#r28641546>.
>